src/dataframe/dataframe.jl

```diff
@@ -721,11 +721,25 @@ Base.hcat(df1::DataFrame, df2::AbstractDataFrame, dfn::AbstractDataFrame...) = h
 ## Missing values support
 ##
 ##############################################################################
+"""
+    allowmissing!(df::DataFrame, col::ColumnIndex)
 
+Convert a single column of a DataFrame from element-type `T` to
```

Comment:
`` `DataFrame` `` and "element type".

I think you could merge the two docstrings into a single one, with one line for each signature (or maybe three, with a line for the single-argument method). Attach it to `function allowmissing! end`.